pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -33,6 +33,16 @@ def load_earth_magnetic_anomaly(
     (see below), and *reg* is grid registration type (**p** for pixel
     registration or **g** for gridline registration).
 
+    The default color palette tables (CPTs) for this dataset are
+    *@earth_mag.cpt* for ``data_source="emag2"`` and
+    ``data_source="emag2_4km"``, and *@earth_wdmam.cpt* for
+    ``data_source="wdmam"``. It's implicitly used when passing in the file name
+    of the dataset to any grid plotting method if no CPT is explicitly
+    specified. When the dataset is loaded and plotted as an
+    :class:`xarray.DataArray` object, the default CPT is ignored and GMT's
+    default CPT (*turbo*) is used. To use the dataset-specific CPT, you need to
+    explicitly set ``cmap="@earth_mag.cpt"`` or ``cmap="@earth_wdmam.cpt"``.
+
```

Comment:
I am wondering whether we should use here "The dataset-specific CPT" instead of "It's", as we have plural in the sentence before? But this is only a detail 🙂.
```suggestion
    The default color palette tables (CPTs) for this dataset are
    *@earth_mag.cpt* for ``data_source="emag2"`` and
    ``data_source="emag2_4km"``, and *@earth_wdmam.cpt* for
    ``data_source="wdmam"``. The dataset-specific CPT is implicitly used when
    passing in the file name of the dataset to any grid plotting method if no
    CPT is explicitly specified. When the dataset is loaded and plotted as an
    :class:`xarray.DataArray` object, the default CPT is ignored and GMT's
    default CPT (*turbo*) is used. To use the dataset-specific CPT, you need to
    explicitly set ``cmap="@earth_mag.cpt"`` or ``cmap="@earth_wdmam.cpt"``.

```